### PR TITLE
add `PutRepositoryTriggers` to `CodeCommit` client

### PR DESCRIPTION
**Pull request by nealio82**
Generated code for `PutRepositoryTriggers` because you wanna know when stuff's happening, right?

Test / Docs examples inspired by:
https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-codecommit-2015-04-13.html#putrepositorytriggers
https://docs.aws.amazon.com/cli/latest/reference/codecommit/put-repository-triggers.html
https://docs.aws.amazon.com/codecommit/latest/APIReference/API_PutRepositoryTriggers.html

### CODE REVIEW

File: src/Service/CodeCommit/tests/Unit/CodeCommitClientTest.php [hunk 1]
```diff
@@ -47,7 +50,7 @@ public function testGetDifferences(): void
         $client = new CodeCommitClient([], new NullProvider(), new MockHttpClient());
 
         $input = new GetDifferencesInput([
-            'repositoryName' => 'change me',
```
Comment (nealio82) [inline src/Service/CodeCommit/tests/Unit/CodeCommitClientTest.php:50]: decided to change this from the lazily-added default while i was here

File: src/Service/CodeCommit/tests/Unit/CodeCommitClientTest.php [hunk 2]
```diff
@@ -47,7 +50,7 @@ public function testGetDifferences(): void
...
+            'repositoryName' => 'my-super-code-repository',
 
             'afterCommitSpecifier' => 'change me',
```
Comment (nealio82) [inline src/Service/CodeCommit/tests/Unit/CodeCommitClientTest.php:55]: ahhh, missed this one though. i'll nuke that one some other time

File: src/Service/CodeCommit/tests/Unit/CodeCommitClientTest.php (hunk 1 above)
Comment (jderusse) [reply src/Service/CodeCommit/tests/Unit/CodeCommitClientTest.php:50]: thank you :heart:

**Comment by jderusse** [comment]
thank you @nealio82

**Comment by jderusse** [comment]
thank you @nealio82

**Pull request merged**